<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettingsGeneralTabWidget</class>
 <widget class="QWidget" name="CameraSettingsGeneralTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>415</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="nx::vms::client::desktop::CameraInfoWidget" name="cameraInfoWidget" native="true"/>
   </item>
   <item>
    <widget class="Line" name="overEditStreamsLine">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::Panel" name="editStreamsPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QPushButton" name="editStreamsButton">
        <property name="text">
         <string>Edit Streams...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="overLicensingLine">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::CameraLicensePanelWidget" name="licensePanel" native="true"/>
   </item>
   <item>
    <widget class="Line" name="overGridLine">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::VirtualCameraUploadWidget" name="virtualCameraUploadWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="spacing">
      <number>24</number>
     </property>
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>16</number>
     </property>
     <item>
      <widget class="QWidget" name="leftWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="nx::vms::client::desktop::ImageControlWidget" name="imageControlWidget" native="true">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="nx::vms::client::desktop::ArchiveLengthWidget" name="virtualCameraArchiveLengthWidget" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="authenticationGroupBox">
          <property name="title">
           <string>Authentication</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="editCredentialsButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Edit Credentials</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>37</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="rightWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,1">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="audioGroupBox">
          <property name="title">
           <string>Audio</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="audioVerticalLayout">
           <item>
            <widget class="QCheckBox" name="enableAudioCheckBox">
             <property name="text">
              <string>Enable audio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="nx::vms::client::desktop::AudioRedirectPickerWidget" name="audioInputRedirectPickerWidget" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>4</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="enableTwoWayAudioCheckBox">
             <property name="text">
              <string>Enable 2-way audio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="nx::vms::client::desktop::AudioRedirectPickerWidget" name="audioOutputRedirectPickerWidget" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="audioOutputDevicePickerGroup" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="nx::vms::client::desktop::VirtualCameraTimeZoneWidget" name="virtualCameraTimeZoneWidget" native="true"/>
        </item>
        <item>
         <widget class="nx::vms::client::desktop::VirtualCameraMotionWidget" name="virtualCameraMotionWidget" native="true"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::Panel</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/panel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ArchiveLengthWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/archive_length_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ImageControlWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/image_control_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::CameraInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/camera_info_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::VirtualCameraMotionWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/virtual_camera_motion_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::CameraLicensePanelWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/camera_license_panel_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::VirtualCameraUploadWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/virtual_camera_upload_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::VirtualCameraTimeZoneWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/virtual_camera_time_zone_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::AudioRedirectPickerWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/widgets/audio_redirect_picker_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>enableAudioCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
